### ISSUE TITLE
[Bug]: 1.11.0: No matching version found for @lit-labs/observers@^3.0.0OBSERVERS

### ISSUE THREAD

**Issue by sandrooco** [sbb-design-systems/lyne-components#3100]
### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/sbb-design-systems/lyne-components/blob/main/docs/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/sbb-design-systems/lyne-components/blob/main/docs/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://github.com/orgs/sbb-design-systems/projects/4) for an issue that matches the one I want to file, without success.

### Bug type

Other

### It affects the following packages

Components

### Which version of Lyne Components are you using

1.11.0

### Operating system

any

### Browser / Browser version

not relevant

### Input mode

None

### What happened?

I'm trying to update to Lyne 1.11.0 (thanks for the fixes!). Unfortunately something seems to be off with the dependencies. When installing them, I get the error "`ETARGET: No matching version found for @lit-labs/observers@^3.0.0OBSERVERS.`" Yarn will let me choose an existing version, npm will just throw an error and exit.

### Steps to reproduce the issue

1. Visit https://stackblitz.com/edit/angular-at-8hg8bk?file=package.json
2. Run `npm install` or `yarn install`
3. The error should appear

### Relevant log output

```shell
❯ npm i
npm ERR! code ETARGET
npm ERR! notarget No matching version found for @lit-labs/observers@^3.0.0OBSERVERS.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.

npm ERR! A complete log of this run can be found in: /home/.npm/_logs/2024-09-20T06_45_39_932Z-debug-0.log
```

**Comment by sandrooco** [comment]
Quick update: I tried adding the following override in my package.json: `"@lit-labs/observers": "2.0.3"` This is the latest version according to npm, which actually works.
So I guess there's just something small wrong with how this dep is handled. :)

**Comment by kyubisation** [comment]
Hello @sandrooco 
Thanks for the bug report. Our dependency assignment was not working as expected and should be fixed by #3102.

**Comment by sandrooco** [comment]
Thank you very much for the quick fix, @kyubisation!
